package.json

```diff
@@ -22,6 +22,7 @@
     "e2e-prepare": "selenium-standalone install",
     "e2e-server": "node ./bin/server.js",
     "version": "npm run build && git add README.md dist/td.js dist/td.min.js",
+    "pretest": "npm run build",
```

Comment:
why have a script that is an alias of another script rather than just call `npm run build` directly before tests?

Reply:
Having built files is a requirement of running the tests, whether in CI or locally.

Reply:
sure, but wouldn't `"test": "npm run build && npm run test:unit"` achieve that and avoid aliasing tasks?

Reply:
🤷‍♀️ This way feels like a cleaner separation of concerns to me but I'm not precious about it